### PR TITLE
Remove a debug directory to save space

### PR DESCRIPTION
**Pull request by S-Dafarra**
Check https://github.com/dic-iit/ify-dependencies-vcpkg/issues/7

How can I know if the packages can be successfully created?

### CODE REVIEW

**Comment by traversaro** [comment]
> How can I know if the packages can be successfully created?

I typically create dummy releases, either in the repo or a personal fork.

**Comment by traversaro** [comment]
Have you tried locally if that works and `find_package(Qt5)` works as well?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -88,6 +89,7 @@ jobs:
         rm -rf ./ify-deps/vcpkg/buildtrees
         rm -rf ./ify-deps/vcpkg/packages
         rm -rf ./ify-deps/vcpkg/downloads
+        rm -rf ./ify-deps/vcpkg/installed/x64-windows-static/tools/qt5/debug
```
Comment (GiulioRomualdi) [inline .github/workflows/ci.yml:92]: This code should be called if the os is `macOS`

Comment (GiulioRomualdi) [reply .github/workflows/ci.yml:92]: Probably you should use this (I'm not sure) 
```suggestion
        rm -rf ./ify-deps/vcpkg/installed/x64-osx/tools/qt5/debug
```

Comment (S-Dafarra) [reply .github/workflows/ci.yml:92]: Thanks!

**Comment by S-Dafarra** [comment]
> Have you tried locally if that works and find_package(Qt5) works as well?

I have tried to rename that folder. It seems to be working fine on a clean build.

**Comment by S-Dafarra** [comment]
Windows failed because of an error during download. Should I simply restart it?

**Comment by traversaro** [comment]
> 
> 
> Windows failed because of an error during download. Should I simply restart it?

Yes, unfortunately that sometimes happens with sourceforge.

**Pull request merged**